core/drivers/crypto/caam/cipher/caam_cipher.c

```diff
@@ -181,8 +181,12 @@ enum caam_status caam_cipher_block(struct cipherdata *ctx, bool savectx,
 	enum caam_status retstatus = CAAM_FAILURE;
 	struct caam_jobctx jobctx = {};
 	uint32_t *desc = ctx->descriptor;
-	struct caamsgtbuf src_sgt = { .sgt_type = false };
```

Comment:
There's no need to change back or forth here, just a clarification for you information.
For one liners this is OK, but I don't see any harm in making it a multiliner either.

Reply:
Alright!